client/jetpack-cloud/sections/settings/advanced-credentials/index.tsx

```diff
@@ -252,7 +253,7 @@ const AdvancedCredentials: FunctionComponent< Props > = ( { action, host, role }
 	const renderConnectedButtons = () => (
 		<>
 			<Button scary disabled={ disableForm } onClick={ handleDeleteCredentials }>
-				{ translate( 'Delete Credentials' ) }
+				{ translate( 'Delete credentials' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.